server/mfAll.js

```diff
@@ -1,513 +1,177 @@
+      "mtime": 1426598155090,
+      "revisionId": "3kLPbjJ9MDAwqGPWe",
+      "_id": "758rqbLQWMqagrxZK"
     },
     "add new course form cover photo label": {
       "key": "add new course form cover photo label",
-      "lang": "en",
-      "text": "Cover Photo",
-      "ctime": 1426503739075,
-      "mtime": 1426503739075,
-      "revisionId": "7jXsYPZKYmuPrzH3j",
-      "file": "./client/templates/teach/teach.html",
-      "line": 12,
-      "template": "teach",
-      "_id": "n77rSvG5CG3SYG9E2"
+      "lang": "fi",
+      "text": "Ikoni",
+      "ctime": 1426598192351,
+      "mtime": 1426598192351,
+      "revisionId": "8h6jjB2a4eyc3Qypd",
+      "_id": "CNjYxPwTyHsaekLGk"
     },
     "add new course form author label": {
       "key": "add new course form author label",
-      "lang": "en",
-      "text": "Author",
-      "ctime": 1426503739075,
-      "mtime": 1426503739075,
-      "revisionId": "LyTj3ghaGdE7HYH5D",
-      "file": "./client/templates/teach/teach.html",
-      "line": 16,
-      "template": "teach",
-      "_id": "B2fz4pJEyWud8M8Sf"
+      "lang": "fi",
+      "text": "Tekijä",
+      "ctime": 1426598205752,
+      "mtime": 1426598205752,
+      "revisionId": "AB5yjR29dQFpyuMXE",
+      "_id": "m2dnb2KK2HaKrL8po"
     },
     "add new course form author placeholder": {
       "key": "add new course form author placeholder",
-      "lang": "en",
-      "text": "Author Name",
-      "ctime": 1426503739075,
-      "mtime": 1426503739075,
-      "revisionId": "ZezaCPddyGecNnhcH",
-      "file": "./client/templates/teach/teach.html",
-      "line": 20,
-      "template": "teach",
-      "_id": "qfTnsiHexX9fGzQGh"
+      "lang": "fi",
+      "text": "Tekijän nimi",
+      "ctime": 1426598213087,
+      "mtime": 1426598213087,
+      "revisionId": "sCnCWPb2J5hN3ycAj",
+      "_id": "WzTecTnZACmNwYuw4"
     },
     "add new course form keywords label": {
       "key": "add new course form keywords label",
-      "lang": "en",
-      "text": "Keywords",
-      "ctime": 1426503739075,
-      "mtime": 1426503739075,
-      "revisionId": "2biCTLSthzmHCniDW",
-      "file": "./client/templates/teach/teach.html",
-      "line": 23,
-      "template": "teach",
-      "_id": "jXxBYNicjeu755AfE"
+      "lang": "fi",
+      "text": "Avainsanat",
+      "ctime": 1426598217323,
+      "mtime": 1426598217323,
+      "revisionId": "NGMNdJFzJWYATtSTX",
+      "_id": "FNyK3m8TyX2Dm3X3h"
     },
     "add new course form keywords placeholder": {
       "key": "add new course form keywords placeholder",
-      "lang": "en",
-      "text": "Choose some tags (e.g. JavaScript, Math)",
-      "ctime": 1426503739075,
-      "mtime": 1426503739075,
-      "revisionId": "d7pSmHWzkofe9Bexd",
-      "file": "./client/templates/teach/teach.html",
-      "line": 27,
-      "template": "teach",
-      "_id": "iwsb4MBomDhN9QimD"
-    },
-    "add new course form published label": {
-      "key": "add new course form published label",
```

Comment:
Keys are still using spaces here. Did this work start from the current development branch? Recent changes to the development branch have fixed these keys.